util/memory/meminfo.go

```diff
@@ -178,3 +179,18 @@ func readUint(path string) (uint64, error) {
 	}
 	return parseUint(strings.TrimSpace(string(v)), 10, 64)
 }
+
+// Get process memory
+func InstanceMemUsed() (uint64, error) {
+	used, t := memUsage.get()
```

Comment:
It seems that we cannot use `memUsage` directly since it returns the `container-mem-usage` instead of the `server-mem-usage`. Please help us confirm this @wshwsh12 
I suggest that we refer to `memUsage` and introduce a new `memInfoCache` named `serverMemUsage`.